event_lead_image/views/event_views.xml

```diff
@@ -0,0 +1,20 @@
+<odoo>
+        <record model="ir.ui.view" id="view_event_form_inherit_image">
+            <field name="name">event.event.form</field>
+            <field name="model">event.event</field>
+            <field name="inherit_id" ref="event.view_event_form" />
+            <field name="arch" type="xml">
+
+                <xpath expr="//div[@class='oe_title']" position="attributes">
```

Comment:
never select classes like this, you can have more than one and the xpath would fail. If you check your instance's logs I bet you'll find a warning for it saying that you should use `hasclass('oe_title')` :wink:

Reply:
@tschanzt what about this?

Reply:
@simahawk ah now i realize what you meant.Somehow i managed to mangle your sentence into you complaining about there possibly being more than one result. I think i used this way quite a lot till now :scream: